src/Infrastructure/Services/Development/NullEmailService.cs

```diff
@@ -1,16 +1,36 @@
-﻿using System;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
 using Application.Contracts;
+using Application.Settings;
+using Microsoft.Extensions.Logging;
 
-namespace Infrastructure.Services.Development
+namespace Infrastructure.Services.Development;
+
+public class NullEmailService : IEmailService
 {
-    public class NullEmailService : IEmailService
+    private readonly ILogger<NullEmailService> _logger;
+    private readonly IEmailService _emailService;
+    private readonly DevSettings _devSettings;
+
+    public NullEmailService(ILogger<NullEmailService> logger, IEmailService emailService, DevSettings devSettings)
```

Comment:
This isn't a `NullEmailService` anymore, it actually does something. Give it a more appropriate name.